scripts/build.js

```diff
@@ -25,7 +25,7 @@ Object.keys(mime).forEach(function (suffix) {
   var s = mime[suffix]
 
   Object.keys(db).forEach(function (type) {
-    if (type.substr(0 - suffix.length) !== suffix) {
+    if (type.slice(0 - suffix.length) !== suffix) {
```

Comment:
I think you can use `-suffix.length` instead of `0 - suffix.length` on this line.

Reply:
You totally can. I have updated the PR